SRTNet.h

```diff
@@ -166,43 +201,49 @@ class SRTNet {
     */
     Mode getCurrentMode() const;
 
-
-    ///Callback handling connecting clients (only server mode)
-    std::function<std::shared_ptr<NetworkConnection>(struct sockaddr& sin, SRTSOCKET newSocket,
-                                                    std::shared_ptr<NetworkConnection>& ctx)> clientConnected = nullptr;
-    ///Callback receiving data type vector
-    std::function<void(std::unique_ptr<std::vector<uint8_t>>& data, SRT_MSGCTRL& msgCtrl,
-                       std::shared_ptr<NetworkConnection>& ctx, SRTSOCKET socket)> receivedData = nullptr;
-
-    ///Callback receiving data no copy
-    std::function<void(const uint8_t* data, size_t size, SRT_MSGCTRL& msgCtrl,
-                       std::shared_ptr<NetworkConnection>& ctx, SRTSOCKET socket)> receivedDataNoCopy = nullptr;
-
-    ///Callback handling disconnecting clients (server and client mode)
-    std::function<void(std::shared_ptr<NetworkConnection> &ctx,
-                                                     SRTSOCKET lSocket)> clientDisconnected = nullptr;
+    /// Callback handling connecting clients (only server mode)
+    std::function<std::shared_ptr<NetworkConnection>(struct sockaddr& sin,
+                                                     SRTSOCKET newSocket,
+                                                     std::shared_ptr<NetworkConnection>& ctx)>
+        clientConnected = nullptr;
+    /// Callback receiving data type vector
```

Comment:
```suggestion

    /// Callback receiving data type vector
```
Pretty sure this formatting didn't improve readability...